### DISCUSSION TITLE
First Next.js contribution - getting set up issues (HookWebpackError)

### DISCUSSION THREAD

**Discussion by kmdev1** [vercel/next.js#24623] [Help]
Hey there 👋🏻 

I'd really love to contribute and I've just followed the `contributing.md` but it seems that I'm missing something - before I start digging around too much, I was just wondering if anyone else had also experienced this issue?

After following the guide and then running a `yarn build` the error I consistently get is:

`HookWebpackError: aSourceMapConsumer.originalPositionFor is not a function`

**Comment by bobarros** [comment]
There is no build script inside next. This command is when you're using it.

Are sure about what you're trying to accomplish?

These are the available scripts.

```json
"scripts": {
    "lerna": "lerna",
    "dev": "lerna run dev --stream --parallel",
    "dev2": "while true; do yarn --check-files && yarn dev; done",
    "testonly": "jest --runInBand",
    "testheadless": "cross-env HEADLESS=true yarn testonly",
    "testsafari": "cross-env BROWSER_NAME=safari yarn testonly",
    "testfirefox": "cross-env BROWSER_NAME=firefox yarn testonly",
    "testie": "cross-env BROWSER_NAME=\"internet explorer\" yarn testonly",
    "testall": "yarn run testonly -- --ci --forceExit && lerna run --scope @next/codemod test",
    "genstats": "cross-env LOCAL_STATS=true node .github/actions/next-stats-action/src/index.js",
    "pretest": "yarn run lint",
    "git-reset": "git reset --hard HEAD",
    "git-clean": "git clean -d -x -e node_modules -e packages -f",
    "test-take2": "yarn git-reset && yarn git-clean && yarn testall",
    "test": "yarn run testall || yarn run test-take2",
    "lint-typescript": "lerna run typescript",
    "lint-eslint": "eslint . --ext js,jsx,ts,tsx --max-warnings=0",
    "lint-no-typescript": "run-p prettier-check lint-eslint",
    "lint": "run-p lint-typescript prettier-check lint-eslint",
    "lint-fix": "yarn prettier-fix && eslint . --ext js,jsx,ts,tsx --fix --max-warnings=0",
    "prettier-check": "prettier --check .",
    "prettier-fix": "prettier --write .",
    "types": "lerna run types --stream",
    "typescript": "lerna run typescript",
    "prepublish": "lerna run prepublish",
    "publish-canary": "lerna version prerelease --preid canary --force-publish && release --pre --skip-questions",
    "publish-stable": "lerna version --force-publish",
    "lint-staged": "lint-staged",
    "next": "node --trace-deprecation packages/next/dist/bin/next",
    "debug": "node --inspect packages/next/dist/bin/next"
  },
```

**Reply by kmdev1** [reply]
Hey!

Apologies I forgot to add where I was actually seeing it.

This happens in a blank create-next-app that has been updated to reference the local version of Next; rather than the Next.js repo itself.

So my current replication steps are:

- Follow steps in Contributing to get running locally
- Create new blank create-next-app
- Follow steps in Contributing section "Running your own app with locally compiled version of Next.js" within the new CNA.
- `yarn dev` works all okay
- `yarn build` throws the error above

Apologies for not adding these details initially

**Reply by darshkpatel** [reply]
I was recently trying to set up the dev environment on a new machine, faced the same issue

**Reply by nihakue** [reply]
Same here. Using a fresh create-next-app application and following the instructions in the contribution readme. 

Commit in next.js is 30ee9e71b7b05e8074fecb9c5179654448211ce0
